<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>POV_Display</class>
 <widget class="QMainWindow" name="POV_Display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POV_Display</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="StartPushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopPushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayportLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select display port</string>
    </property>
   </widget>
   <widget class="QLabel" name="modeLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select display mode</string>
    </property>
   </widget>
   <widget class="QComboBox" name="displayComboBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="modeComboBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Text</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Image</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="runtimeLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter running time (sec)</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayhintLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter display text</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="TimerSpinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLabel" name="dcmportLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select DC motor port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dcmComboBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
